src/engine/controls/loopingcontrol.cpp

```diff
@@ -138,6 +138,11 @@ LoopingControl::LoopingControl(const QString& group,
     m_pCOBeatLoopActivate = new ControlPushButton(ConfigKey(group, "beatloop_activate"));
     connect(m_pCOBeatLoopActivate, &ControlObject::valueChanged,
             this, &LoopingControl::slotBeatLoopToggle);
+    m_pCOBeatLoopKeepLoopIn = new ControlPushButton(ConfigKey(group, "beatloop_keep_loopin"));
```

Comment:
```suggestion
    m_pCOBeatLoopKeepLoopIn = new ControlPushButton(ConfigKey(group, "beatloop_keep_loop_in"));
```
To match the "loop_in" control.